script.js

```diff
@@ -22,3 +22,223 @@ navLinksEl.forEach((item) => {
+    FeaturedImage: 'images/Snapshoot Portfolio.png',
+  },
+
+  {
+    Name: 'Uber Navigation',
+    Description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+    FeaturedImage: 'images/Snapshoot Portfolio (3).png',
+  },
+];
+
+const sectionHide = document.querySelector('.container');
+const btnEl1 = document.querySelector('.btn-one');
+const btnEl2 = document.querySelector('.btn-two');
+const btnEl3 = document.querySelector('.btn-three');
+const btnEl4 = document.querySelector('.btn-four');
+const body = document.querySelector('body');
+
+btnEl1.addEventListener('click', () => {
+  sectionHide.classList.toggle('hide-container');
+  body.classList.toggle('background-img');
+  const Popsection = document.createElement('section');
+  Popsection.className = 'Popup-Section';
+  Popsection.innerHTML = `
+      <div class="icon-holder">
+      <h2>${workCards[0].Name} </h2>
+      <a onclick="showMenu(close)"><i class="fa-solid fa-xmark close-cards"></i></a>
+      </div>
+      <article class="wrap-frame">
+      <p class="frame canopy">CANOPY</p>
+      <img src="images/Counter.png" alt="">
+      <p class="frame">Back End Dev</p>
+      <img class="frame" src="images/Counter.png" alt="" class="card-img">
+      <p class="frame">2015</p>
+      </article>
+      <img src= "${workCards[0].FeaturedImage}" alt="Snapshoot" class="card-img desk-img"> 
+      <div class="right-pop">
+      <p class="par pop-par">${workCards[0].Description} </p>
+      <div class="column-pop">
+      <div class="lang-button">
+      <ul>
+          <li><a href="#">${workCards[0].Technologies[0]}</a></li>
+          <li> <a href="#">${workCards[0].Technologies[1]}</a></li>
+          <li><a href="#">${workCards[0].Technologies[2]}</a></li>
+      </ul>
```

Comment:
- [ ] Kindly try to decrease the margin and padding between the li languages to be like the Figma Design 😃
- [ ] Also, consider adding the ```hr``` which is below the ul languages to be like the Figma design and separate them from the btns of the see project 😃
- [ ] Also kindly try to stick to the color of  ```a``` in the Figma Design to make your popup amazing for the visitors😊

> Check below to see the difference 😊⤵️

| Figma design | Your design| 
| --- | --- |
|   ![image](https://user-images.githubusercontent.com/83360636/216099624-3a21106c-cb15-4206-87a6-4eedba41f811.png)| ![image](https://user-images.githubusercontent.com/83360636/216099678-693dc49d-787c-468a-b558-24a5d02e9207.png)|